presto-tpch/src/main/java/com/facebook/presto/tpch/TpchMetadata.java

```diff
@@ -101,6 +112,15 @@ public TpchMetadata(String connectorId, boolean predicatePushdownEnabled, Column
         this.connectorId = connectorId;
         this.predicatePushdownEnabled = predicatePushdownEnabled;
         this.columnNaming = columnNaming;
+        this.statisticsEstimator = createStatisticsEstimator();
+    }
+
+    private static StatisticsEstimator createStatisticsEstimator()
+    {
+        ObjectMapper objectMapper = new ObjectMapper()
+                .registerModule(new Jdk8Module());
+        TableStatisticsDataRepository tableStatisticsDataRepository = new TableStatisticsDataRepository(objectMapper);
```

Comment:
Probably it could, but I prefer passing dependencies to objects than making them create instantiate themselves.